use @g_io_error_from_errno[U32](err_no': I32)
